src/solver/conv_hip_implicit_gemm_fwd_xdlops.cpp

```diff
@@ -227,6 +226,40 @@ void RunCKSolution(const Handle& handle,
     }
 }
 
+InvokerFactory MakeInvokerFactoryHipImplGemmFwdXdlops(const ProblemDescription& problem, 
+    const PerformanceConfigHipImplicitGemmFwdXdlops& config) {
+
+  auto ck_args = CKArgs{problem};
+  miopenDataType_t data_type = problem.conv_problem.GetInDataType();
+  auto config_idx = config.index;
+
+
+  InvokerFactory ret = [ck_args, data_type, config_idx] (const std::vector<Kernel>& kernels) {
```

Comment:
[R] To simplify the code.
```suggestion
  InvokerFactory ret = [=] (const std::vector<Kernel>& kernels) {
```

Reply:
It depends on how explicitly we want to capture the values. Defaulted `by value` capture may lead to the problem if someone blindly capture a vector or any other heavy-copied class. It's just available in the lambda due to defaulted behavior.

Reply:
@CAHEK7 Very good observation! [Resolved]